uber/models.py

```diff
@@ -961,6 +970,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     no_shirt          = relationship('NoShirt', backref=backref('attendee', load_on_pending=True), uselist=False)
     admin_account     = relationship('AdminAccount', backref=backref('attendee', load_on_pending=True), uselist=False)
     food_restrictions = relationship('FoodRestrictions', backref=backref('attendee', load_on_pending=True), uselist=False)
+    watchlist_entry   = relationship('WatchList', backref=backref('attendee', load_on_pending=True), uselist=False)
```

Comment:
See my comment below about the direction of the foreign key; I suspect that we're defining this in the wrong direction.